cirq/google/sim/xmon_simulator.py

```diff
@@ -133,15 +143,20 @@ def _run(
         repetitions: int,
     ) -> Dict[str, List[np.ndarray]]:
         """See definition in `cirq.SimulatesSamples`."""
-        xmon_circuit, keys = self._to_xmon_circuit(
-            circuit,
-            param_resolver)
-        if xmon_circuit.are_all_measurements_terminal():
-            return self._run_sweep_sample(xmon_circuit, repetitions)
+
```

Comment:
update doc describing the verification and throwing of error

Reply:
Done.